### PR TITLE
doc: noted ngx.req.get_method also can be used in the body_filter_lua* phase and log_by_lua* phase

### PR DESCRIPTION
**Pull request by tokers**
I hereby granted the copyright of the changes in this pull request
to the authors of this lua-nginx-module project.

### CODE REVIEW

**Comment by tokers** [comment]
@agentzh @thibaultcha

File: doc/HttpLuaModule.wiki [hunk 1]
```diff
@@ -3563,7 +3563,7 @@ This method does not work in HTTP/2 requests yet.
 == ngx.req.get_method ==
 '''syntax:''' ''method_name = ngx.req.get_method()''
 
-'''context:''' ''set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, balancer_by_lua*''
+'''context:''' ''set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, balancer_by_lua*, log_by_lua*''
```
Comment (thibaultcha) [inline doc/HttpLuaModule.wiki]: For completeness' sake, we should also note that this API works in `body_filter_by_lua*` as well.

Comment (tokers) [reply doc/HttpLuaModule.wiki]: OK. I will also note this.

**Comment by tokers** [comment]
@thibaultcha Updated.

**Comment by thibaultcha** [comment]
@tokers Thank you

**Pull request merged**